docs/getting-help.rst

```diff
@@ -93,148 +98,140 @@ Please provide *all* of the following information:
    reproducing the problem.  This will allow the Open MPI developers
    to see the error for themselves, and therefore be able to fix it.
 
+Please provide *all* of the following information:
+
 #. The version of Open MPI that you're using.
 
-#. The ``config.log`` file from the top-level Open MPI directory, if
-   available (**compress or post to a Github gist or Pastebin**).
+#. The stdout and stderr from running ``configure``.
 
-#. The output of the ``ompi_info --all`` command from the node where
-   you're invoking ``mpirun``.
+#. All ``config.log`` files from the Open MPI build tree.
 
-#. If you have questions or problems about process affinity /
-   binding, send the output from running the ``lstopo -v``
-   command from a recent version of `Hwloc
-   <https://www.open-mpi.org/projects/hwloc/>`_.  *The detailed
-   text output is preferable to a graphical output.*
+#. Output from when you ran ``make V=1 all`` to build Open MPI.
 
-#. If running on more than one node |mdash| especially if you're
-   having problems launching Open MPI processes |mdash| also include
-   the output of the ``ompi_info --version`` command **from each node
-   on which you're trying to run**.
+#. Output from when you ran ``make install`` to install Open MPI.
 
-   #. If you are able to launch MPI processes, you can use
-      ``mpirun`` to gather this information.  For example, if
-      the file ``my_hostfile.txt`` contains the hostnames of the
-      machines on which you are trying to run Open MPI
-      processes::
+You can use a sequence of commands similar to the following to gather
+much of the above information (adjust as necessary for your specific
+environment):
 
-         shell$ mpirun --map-by node --hostfile my_hostfile.txt --output tag ompi_info --version
+.. code-block:: sh
 
+   # Make a directory for the output files, then build/install Open MPI
+   shell$ mkdir ompi-output
+   shell$ ./configure {options} 2>&1 | tee ompi-output/config.out
+   shell$ tar cf - `find . -name config.log` | tar -C ompi-output -x -
+   shell$ make V=1 all 2>&1          | tee ompi-output/make.out
+   shell$ make install 2>&1          | tee ompi-output/make-install.out
```

Comment:
Suggestion: Remove the `shell$ ` from the output, so users can copy/paste this into their terminal.

Reply:
That's a fair point, but we used `shell$` consistently throughout the docs to indicate that these are commands you type at a shell prompt.

Here's a compromise: I'll turn these into shell scripts, and therefore `shell$` isn't relevant.

Reply:
Sounds fine. Elsewhere I think it's fine to have the `shell$ ` as an indicator. These help pages seem special, though. In that, we want the user to give us (nearly) exactly what those commands are doing so we have all of the information necessary to help them. So if the user can copy/paste them easily, then it is more likely that they will do so and not miss a step.